datalad/distribution/tests/test_install.py

```diff
@@ -60,7 +65,13 @@ def test_installationpath_from_url():
               'http://example.com/lastbit',
               'http://example.com/lastbit.git',
               ):
-        assert_equal(_installationpath_from_url(p), 'lastbit')
+        assert_equal(_get_installationpath_from_url(p), 'lastbit')
+
+
+def test_get_git_url_from_source():
+    eq_(_get_git_url_from_source('///subds'), DATASETS_TOPURL + 'subds')
+    eq_(_get_git_url_from_source('http://example.com'), 'http://example.com')
+    assert_raises(NotImplementedError, _get_git_url_from_source, '//custom/subds')
 
```

Comment:
Yes, we should try. Or let's say: test. ;)
Any chance to just add two assertions or did get something wrong?